src/components/StakingVaults/DeFiEarn.tsx

```diff
@@ -38,14 +38,7 @@ export const DeFiEarn: React.FC<DefiEarnProps> = ({
     <Flex width='full' flexDir='column' gap={6}>
       {header && header}
       <Tabs isLazy variant='soft-rounded' size='sm'>
```

Comment:
First mount is expected to be slower since this is lazy-loaded but can we make it so that toggling between by asset and by provider isn't consistently unmounting/remounting?

```suggestion
      <Tabs lazyBehavior='keepMounted' isLazy variant='soft-rounded' size='sm'>
```

Tested locally, that makes subsequent toggles blazingly faster - from a user perspective, if I go to one tab as a user, I may want to go back and forth so that behavior might make more sense, your call